src/test/java/com/chikli/hudson/plugin/naginator/NaginatorCauseTest.java

```diff
@@ -156,42 +158,44 @@ public void testCauseLinkWithLargeNumber() throws Exception {
 
         assertEquals(2002, p.getLastBuild().getNumber());
 
-        WebClient wc = j.createWebClient();
-        {
-            FreeStyleBuild b = p.getBuildByNumber(2002);
-            FreeStyleBuild causeBuild = p.getBuildByNumber(2001);
-            HtmlPage page = wc.getPage(b);
-            HtmlAnchor anchor = page.getFirstByXPath("//a[contains(@class,'naginator-cause')]");
-            assertNotNull(anchor);
-            assertEquals(
-                getRootURL() + causeBuild.getUrl(),
-                anchor.getHrefAttribute()
-            );
-        }
-        {
-            FreeStyleBuild b = p.getBuildByNumber(2001);
-            FreeStyleBuild causeBuild = p.getBuildByNumber(2000);
-            HtmlPage page = wc.getPage(b);
-            HtmlAnchor anchor = page.getFirstByXPath("//a[contains(@class,'naginator-cause')]");
-            assertNotNull(anchor);
-            assertEquals(
-                getRootURL() + causeBuild.getUrl(),
-                anchor.getHrefAttribute()
-            );
+        try (WebClient wc = j.createWebClient()) {
+            {
+                FreeStyleBuild b = p.getBuildByNumber(2002);
+                FreeStyleBuild causeBuild = p.getBuildByNumber(2001);
+                HtmlPage page = wc.getPage(b);
+                HtmlAnchor anchor = page.getFirstByXPath("//a[contains(@class,'naginator-cause')]");
+                assertNotNull(anchor);
+                assertEquals(
+                        getRootURL() + causeBuild.getUrl(),
+                        anchor.getHrefAttribute()
+                );
+            }
+            {
+                FreeStyleBuild b = p.getBuildByNumber(2001);
+                FreeStyleBuild causeBuild = p.getBuildByNumber(2000);
+                HtmlPage page = wc.getPage(b);
+                HtmlAnchor anchor = page.getFirstByXPath("//a[contains(@class,'naginator-cause')]");
+                assertNotNull(anchor);
+                assertEquals(
+                        getRootURL() + causeBuild.getUrl(),
+                        anchor.getHrefAttribute()
+                );
+            }
         }
     }
 
     @Issue("SECURITY-2946")
     @Test
-    public void testEscapedDisplayname() throws Exception {
+    public void testEscapedDisplayName() throws Exception {
         FreeStyleProject p = j.createFreeStyleProject();
         FreeStyleBuild build1 = p.scheduleBuild2(0).get();
-        build1.setDisplayName​("<div id=\"unescaped-displayname\">bad displayname</div>");
+        build1.setDisplayName("<div id=\"unescaped-displayname\">bad displayname</div>");
```

Comment:
Change note: Unprintable whitespace between `setDisplayName` and `(` is removed.